lib-ee/emqx_license/src/emqx_license_schema.erl

```diff
@@ -34,55 +32,13 @@ roots() ->
 
 fields(key_license) ->
     [
-        {type, #{
-            type => key,
-            required => true,
-            desc => ?DESC(license_type_field)
-        }},
         {key, #{
```

Comment:
Please move the default value from config file to here.
(a requirement from cloud-native team, for smooth upgrade from opensource edition to enterprise edition).

Reply:
:heavy_check_mark: